cmd/dextre/cmd/common.go

```diff
@@ -0,0 +1,24 @@
+package cmd
+
+import (
+	"log"
+)
+
+func checkGoldenFile() {
```

Comment:
There's no return value. How do we know the check succeeds?

Reply:
If the check fails, execution stops.

Reply:
Doesn't it look like out of control? While reading the command I have no idea of what's going to happen or how will this behave.

Reply:
If you return an error, then you can wrap the calls to this method with `must()`:
```
func must(err error) {
        if err != nil {
                log.Fatal(err)
        }
}
```